### ISSUE TITLE
无法安装

### ISSUE THREAD

**Issue by DLSP12138** [iwantavnow/Setting.App#2]
电脑显示这个提示窗App installed on your other devices will stop working. Are you sure you want to continue?，之后手机没有安装进软件。怎么解决

**Comment by iwantavnow** [comment]
By this step, it means that you have completed the registration on the Apple side.
The next step is to automatically download the latest version, then sign and install it.
The cause of the error should be that the address of the latest version is **blocked by the Internet**. 
We will provide more areas in the **next version to install**

**Comment by DLSP12138** [comment]
Thank you. I hope this software can be better.

**Comment by iwantavnow** [comment]
Duplicate of #1
